# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxMacFont.h
# Commit: bf0dd96b4423
# Full Hash: bf0dd96b442327acd6df0db0f80886a53a4cec52
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 7 - Route COLRv1 fonts on macOS through our COLR-rendering path; only COLRv0 can use the native Core Text support. r=gfx-reviewers,lsalzman
#   
#   Depends on D153880
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152162
# ==============================================================================

diff -r d0a5d11277bb -r bf0dd96b4423 gfx/thebes/gfxMacFont.h
--- a/gfx/thebes/gfxMacFont.h	Wed Aug 17 10:59:35 2022 +0000
+++ b/gfx/thebes/gfxMacFont.h	Wed Aug 17 10:59:35 2022 +0000
@@ -49,6 +49,8 @@
 
   FontType GetType() const override { return FONT_TYPE_MAC; }
 
+  bool UseNativeColrFontSupport() const override;
+
   // Helper to create a CTFont from a CGFont, with optional font descriptor
   // (for features), and copying any variations that were set on the CGFont.
   // This is public so that gfxCoreTextShaper can also use it.